pages/price-feeds/migrate-an-app-to-pyth/chainlink.md

```diff
@@ -0,0 +1,100 @@
+# Migrate from Chainlink to Pyth
+
+This guide explains how to migrate an EVM application that uses Chainlink price feeds to use Pyth price feeds.
+Pyth provides a Chainlink-compatible interface for its price feeds to make this process simple.
+There are two main steps to the migration:
+
+1. Deploy the `PythAggregatorV3` contract to provide a Chainlink-compatible feed interface.
+2. Schedule price updates for the feeds required by your app.
+
+## Install Pyth SDKs
+
+The `PythAggregatorV3` contract is provided in the [Pyth Price Feeds Solidity SDK](https://github.com/pyth-network/pyth-crosschain/tree/main/target_chains/ethereum/sdk/solidity).
+Add this SDK to the dependencies of your EVM contract.
+
+**Truffle/Hardhat**
+
+If you are using Truffle or Hardhat, simply install the NPM package:
+
+```bash copy
+npm install @pythnetwork/pyth-sdk-solidity
+```
+
+**Foundry**
+
+If you are using Foundry, you will need to create an NPM project if you don't already have one.
+From the root directory of your project, run:
+
+```bash copy
+npm init -y
+npm install @pythnetwork/pyth-sdk-solidity
+```
+
+Then add the following line to your `remappings.txt` file:
+
+```text copy
+@pythnetwork/pyth-sdk-solidity/=node_modules/@pythnetwork/pyth-sdk-solidity
+```
+
+## Deploy Adapter Contract
+
+Chainlink-compatible applications can deploy and use `PythAggregatorV3` in `@pythnetwork/pyth-sdk-solidity` as a replacement for their Chainlink price feeds.
+`PythAggregatorV3` is an adapter contract that wraps the Pyth contract and implements Chainlink's `AggregatorV3Interface`.
+
+One important difference between Pyth and Chainlink is that the Pyth contract holds data for all price feeds, whereas Chainlink has separate instances of `AggregatorV3Interface` for each feed.
```

Comment:
This narrative feels better.

"Pyth contract holds data for all price feeds, in comparison to Chainlink, which has separate instances of AggregatorV3Interface` for every feed.